codesamples/factories.py

```diff
@@ -27,11 +27,11 @@ def initial_data():
         (
             """\
             <pre><code><span class=\"comment\"># Simple output (with Unicode)</span>
-            >>> print(\"Hello, I'm Python!\")
+            >>> print("Hello, I'm Python!")
             <span class=\"output\">Hello, I'm Python!</span>
```

Comment:
Shall we put a unicode snake into the code (and HTML)?

```suggestion
            >>> print("Hello, I'm Python \N{SNAKE}")
            <span class=\"output\">Hello, I'm Python! &#x1f40d;</span>
```

![image](https://user-images.githubusercontent.com/55152140/205651672-524c15e1-3fbf-486c-804f-52949a375c0f.png)